src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/detentionengagementteam/DetentionEngagementTeamRequestResponseReviewPersonalisation.java

```diff
@@ -71,9 +73,8 @@ public String getReferenceId(Long caseId) {
     public Map<String, Object> getPersonalisationForLink(AsylumCase asylumCase) {
         requireNonNull(asylumCase, "asylumCase must not be null");
 
-        String hearingDate = LocalDate.parse(asylumCase.read(LIST_CASE_HEARING_DATE, String.class)
-                .orElseThrow(() -> new IllegalStateException("Hearing date is not present")))
-            .format(DateTimeFormatter.ofPattern("d MMM yyyy"));
+        String hearingDate = dateTimeExtractor.extractHearingDate(asylumCase.read(LIST_CASE_HEARING_DATE, String.class)
+            .orElseThrow(() -> new IllegalStateException("Hearing date is not present")));
```

Comment:
Shouldnt this be
dateTimeExtractor.extractHearingDate(hearingDetailsFinder.getHearingDateTime(asylumCase))

I believe the hearingDetailsFinder.getHearingDateTime method already throws the exception too.

Reply:
I didn't use the HearingDetailsFinder::getHaringDateTime. Maybe I should use that.

Reply:
I have taken care of this by using  HearingDetailsFinder::getHaringDateTime to get the hearing date. Thanks.